test/xUnit/csharp/test_RemotingUtils.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Management.Automation.Remoting;
+using Microsoft.PowerShell.Commands;
+using Xunit;
+
+namespace PSTests.Parallel
+{
+    public class RemotingUtilsTests
+    {
+        private readonly NewPSSessionCommand cmdlet;
+
+        public RemotingUtilsTests()
+        {
+            cmdlet = new NewPSSessionCommand();
+        }
+
+        [Fact]
+        public void TestParseSshHostname()
+        {
+            string hostname = "foo.bar.com";
+            RemotingUtils.ParseSshHostName(cmdlet, hostname, out string host, out string userName, out int port);
+            Assert.Equal(hostname, host);
+            Assert.Null(userName);
+            Assert.Equal(0, port);
+        }
+
+        [Fact]
+        public void TestParseSshHostnameWithPort()
+        {
+            string hostname = "foo.bar.com:22";
```

Comment:
Test would be slightly better if you had some upper case characters to validate it is preserved